frontstage/templates/help/help-info-ons.html

```diff
@@ -5,12 +5,8 @@
 {% from "components/button/_macro.njk" import onsButton %}
 {% set breadcrumbsData = [
     {
-        "text": "Sign in",
-        "url": "/sign-in"
-    },
-    {
-        "text": "Help",
-        "url": "/help"
+        "text": "Back",
+        "url": "/surveys/todo"
```

Comment:
Incase you click the link when you are on the completed tab and load the info about ONS, then shouldn't this return back to he completed tab?

Reply:
Ideally yes, but we need a rework of all the back links and navigation to make them dynamic going forward. For mvp the navigation of the back links to todo is the simplest, otherwise we will have to loop back and update all breadcrumbs to make sure they play well together.

Reply:
With the test, i didn't think we tested on breadcrumbs, just on the loading of the resulting page which is covered in the testing for info-ons. I do need to remove some of the views though from those pages so, which i'll do now. Coverage should go up as a result.

Reply:
I am a bit confused as to why all the testing for the help pages are named sign_out_help, it might be worth revisiting all this afterwards? especially after removing some of the pages as some of the naming and tests are a bit strange